sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -388,20 +391,29 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
       val db1 = "spark_29425_1"
       val db2 = "spark_29425_2"
       val owner = "spark_29425"
+      val currentUser = Utils.getCurrentUserName()
 
       sql(s"CREATE DATABASE $db1")
-      checkOwner(db1, Utils.getCurrentUserName())
+      checkOwner(db1, currentUser, "USER")
       sql(s"ALTER DATABASE $db1 SET DBPROPERTIES ('a'='a')")
-      checkOwner(db1, Utils.getCurrentUserName())
-
-      // TODO: Specify ownership should be forbidden after we implement `SET OWNER` syntax
-      sql(s"CREATE DATABASE $db2 WITH DBPROPERTIES('ownerName'='$owner')")
-      checkOwner(db2, owner)
-      sql(s"ALTER DATABASE $db2 SET DBPROPERTIES ('a'='a')")
-      checkOwner(db2, owner)
-      // TODO: Changing ownership should be forbidden after we implement `SET OWNER` syntax
-      sql(s"ALTER DATABASE $db2 SET DBPROPERTIES ('ownerName'='a')")
-      checkOwner(db2, "a")
+      checkOwner(db1, currentUser, "USER")
+      val e = intercept[AnalysisException](sql(s"ALTER DATABASE $db1 SET DBPROPERTIES ('a'='a',"
+        + s"'ownerName'='$owner','ownerType'='XXX')"))
```

Comment:
does case sensitivity matter for reserved properties? what if users specify `Ownername`?

Reply:
ok maybe it's fine to treat `Ownername` as a normal property.

Reply:
It does not matter but I guess we should mention this

```
spark-sql> create namespace abcde with properties('LOCATION'= 'b');
20/01/09 17:21:54 INFO HiveMetaStore: 0: get_database: global_temp
20/01/09 17:21:54 INFO audit: ugi=kentyao	ip=unknown-ip-addr	cmd=get_database: global_temp
20/01/09 17:21:54 WARN ObjectStore: Failed to get database global_temp, returning NoSuchObjectException
20/01/09 17:21:54 INFO HiveMetaStore: 0: create_database: Database(name:abcde, description:, locationUri:file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/spark-warehouse/abcde.db, parameters:{LOCATION=b}, ownerName:kentyao, ownerType:USER)
20/01/09 17:21:54 INFO audit: ugi=kentyao	ip=unknown-ip-addr	cmd=create_database: Database(name:abcde, description:, locationUri:file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/spark-warehouse/abcde.db, parameters:{LOCATION=b}, ownerName:kentyao, ownerType:USER)
20/01/09 17:21:54 WARN ObjectStore: Failed to get database abcde, returning NoSuchObjectException
20/01/09 17:21:54 INFO FileUtils: Creating directory if it doesn't exist: file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/spark-warehouse/abcde.db
Time taken: 1.891 seconds
```

```
spark-sql> create namespace abcdef with properties('location'= 'b');
20/01/09 17:22:52 INFO HiveMetaStore: 0: create_database: Database(name:abcdef, description:, locationUri:file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/b, parameters:{}, ownerName:kentyao, ownerType:USER)
20/01/09 17:22:52 INFO audit: ugi=kentyao	ip=unknown-ip-addr	cmd=create_database: Database(name:abcdef, description:, locationUri:file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/b, parameters:{}, ownerName:kentyao, ownerType:USER)
20/01/09 17:22:52 WARN ObjectStore: Failed to get database abcdef, returning NoSuchObjectException
20/01/09 17:22:52 INFO FileUtils: Creating directory if it doesn't exist: file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/b
```

Reply:
```
spark-sql> desc namespace extended abcde;
20/01/09 17:24:14 INFO HiveMetaStore: 0: get_database: abcde
20/01/09 17:24:14 INFO audit: ugi=kentyao	ip=unknown-ip-addr	cmd=get_database: abcde
20/01/09 17:24:14 INFO HiveMetaStore: 0: get_database: abcde
20/01/09 17:24:14 INFO audit: ugi=kentyao	ip=unknown-ip-addr	cmd=get_database: abcde
Database Name	abcde
Description
Location	file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/spark-warehouse/abcde.db
Owner Name	kentyao
Owner Type	USER
Properties	((LOCATION,b))
Time taken: 0.048 seconds, Fetched 6 row(s)
20/01/09 17:24:14 INFO SparkSQLCLIDriver: Time taken: 0.048 seconds, Fetched 6 row(s)
spark-sql> desc namespace extended abcdef;
20/01/09 17:24:21 INFO HiveMetaStore: 0: get_database: abcdef
20/01/09 17:24:21 INFO audit: ugi=kentyao	ip=unknown-ip-addr	cmd=get_database: abcdef
20/01/09 17:24:21 INFO HiveMetaStore: 0: get_database: abcdef
20/01/09 17:24:21 INFO audit: ugi=kentyao	ip=unknown-ip-addr	cmd=get_database: abcdef
Database Name	abcdef
Description
Location	file:/Users/kentyao/Downloads/spark/spark-3.0.0-SNAPSHOT-bin-20200103/b
Owner Name	kentyao
Owner Type	USER
Properties
Time taken: 0.016 seconds, Fetched 6 row(s)
```